src/mlpack/tests/main.cpp

```diff
@@ -8,10 +8,7 @@
  * 3-clause BSD license along with mlpack.  If not, see
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
-#ifndef MLPACK_ENABLE_ANN_SERIALIZATION
-  #define MLPACK_ENABLE_ANN_SERIALIZATION
```

Comment:
are we doing this manually now? would not this affect the examples?

Reply:
No, it's still necessary to define `MLPACK_ENABLE_ANN_SERIALIZATION` for serializing neural networks, but this isn't needed for `main.cpp`, which is in a separate translation unit from the other `.cpp` files.  Instead it is only needed directly in the files that test neural network serialization.  This helps speed up compilation of those files that don't need neural network serialization.